src/components/common/EntityInfo.vue

```diff
@@ -109,8 +108,8 @@ export default class EntityInfo extends Mixins(EnumMixin, DateMixin) {
   /** The entity description.  */
   get entityDescription (): string {
     const corpTypeDescription = this.getCorpTypeDescription(this.getEntityType)
-    if (this.isTypeSoleProp) {
-      return `${corpTypeDescription} / Doing Business As (DBA) ${this.getFilingName}`
+    if (this.isTypeFirm) {
+      return `${corpTypeDescription}`
```

Comment:
What is the requirement here?

Reply:
I was just advised to remove the DBA title from the header in the Dissolution Dashboard for SP

Reply:
For dissolutions only? This header is also used for firm registrations.